extensions/amp-intaker-chat-widget/0.1/cookies.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const CookiesAPI = {
```

Comment:
This will need a comment

Reply:
As far as I know there is no cookie support in AMP! right?
This is a fully working widget in native js for normal sites and I did not wanted to maintain 2 different versions for normal sites and AMP sites, so I made some changes to make it work with AMP, in normal sites CookiesAPI is loaded from customized https://github.com/js-cookie/js-cookie.

I added the empty CookiesAPI code to make the widget work as is.

Reply:
updated, now using the cookie module, I am gana need a `Privacy Review`